presto-product-tests/src/main/resources/sql-tests/testcases/session_functions/currentUserAccessible.result

```diff
@@ -0,0 +1,2 @@
+-- delimiter: |; ignoreOrder: true; ignoreExcessRows: true; trimValues:true
+user
```

Comment:
I suggest you remove this product test. It does not add extra value compared with unit tests you already added.